### ISSUE TITLE
Question about how to use elucidated_diffusion

### ISSUE THREAD

**Issue by cyril-data** [lucidrains/denoising-diffusion-pytorch#308]
I apologize in advance for this very naive question, but could you possibly provide an example (or the basics) of using `elucidated_diffusion`, similar to how you provided an example of using `Unet` and `GaussianDiffusion` from the `denoising_diffusion_pytorch` library in the readme ?

**Comment by AlejandroSantorum** [comment]
I agree with the comment above. It would be great to have some examples or documentation about how to use each of the replicated models. Could you add some to the README @lucidrains ? 
I would love to see an example on how to use the new model in `repaint.py`, specially on how to use the ground-truth image and a mask to reconstruct the masked region as in the original paper.

Thank you for your time :)
